.github/workflows/e2e.yaml

```diff
@@ -0,0 +1,31 @@
+name: Run tests
+
+on: [ push, pull_request ]
+
+permissions:
+  contents: read
+
+jobs:
+  e2e:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Install Go
+      uses: actions/setup-go@v4
+      with:
+        go-version: 1.22.6
+    - name: Checkout code
+      uses: actions/checkout@v3
+    - name: Update vendor modules
+      run: go mod vendor
```

Comment:
The "go mod vendor" step shouldn't be necessary.

Or does building containers need it?

Reply:
I copied the template for test so I'm not sure if it is needed or not.

Reply:
@kannon92 you can remove it and see if the job survives.

Reply:
done. let us see.

Reply:
guess we don't!